src/webui/www/private/scripts/client.js

```diff
@@ -752,7 +752,7 @@ window.addEventListener("DOMContentLoaded", function() {
                             const torrents = response['trackers'][tracker];
                             const hash = window.qBittorrent.Client.genHash(getHost(tracker));
 
-                            // the reason why we need the merge here is because the web ui api returned trackers may have different url for the same tracker host.
+                            // the reason why we need the merge here is because the WebUI api returned trackers may have different url for the same tracker host.
```

Comment:
```suggestion
                            // the reason why we need the merge here is because the WebUI API returned trackers may have different URL for the same tracker host.
```

Reply:
Although I certainly see that all the comments around use the wrong case in the same cases, so I don't insist.